p2p/protocol/holepunch/coordination.go

```diff
@@ -174,7 +174,7 @@ func (hs *Service) beginDirectConnect(p peer.ID) error {
 // It first attempts a direct dial (if we have a public address of that peer), and then
 // coordinates a hole punch over the given relay connection.
 func (hs *Service) DirectConnect(p peer.ID) error {
-	log.Debugw("got inbound proxy conn from peer", p)
+	log.Debugw("got inbound proxy conn", "peer", p.Pretty())
```

Comment:
No need to call `Pretty()`, the peer ID will get encoded on-demand if debug logging is enabled.

Reply:
Fixed.